src/components/SEO/index.tsx

```diff
@@ -72,6 +72,13 @@ const SEO: React.FC<Props> = ({ description, lang, title }) => {
           content: metaDescription
         }
       ]}
+      script={[
+        {
+          src:
+            'https://unpkg.com/@debtcollective/dc-header-component@0.12.2/dist/web-header/web-header.js',
```

Comment:
@arathjz I think we should bundle this components instead of using unpkg.

Reply:
I guess you mean us to include the minified version of the distribution on our code _(AKA, `public/` folder)_ I wonder if that will give us hard times in the future as the browser may cache the request to this file as we are not going to benefit from gatsby including content hashes due to browsers caching.

Reply:
Also, having a CDN as `unpkg.com` will allow us to target `@latest` version and ship updates faster.

Reply:
I mean you can add the dependency to the package.json and import it, we don't have to use it this way. `unpkg` comes handy for example in the old landing page where we don't have a modules like we do in this gatsby app.

I'm not sure but it should just be

```
yarn add @debtcollective/header-component

import Header from "@debtcollective/header-component";
```

Reply:
Agree, it would be better to install it as a dependency. I'd say let's add this change on another PR since may need some changes on the header component